src/consts/icons.ts

```diff
@@ -29,6 +29,12 @@ export enum EIconName {
   SHARE = 'share',
   SIMPLE_SHARE = 'simple_share',
   HAND_COINS = 'hand_coins',
+  COINS_PURPLE = 'coins_purple',
```

Comment:
would also remove the purple from the names

Reply:
done